integration_tests/testsuite/deploy_app.py

```diff
@@ -64,7 +64,7 @@ def _record_latency_to_bigquery(deploy_latency, language):
 
 
 def deploy_app(base_image, builder_image, appdir,
-               yaml, record_latency, language):
+               yaml, record_latency = False, language = None):
```

Comment:
nit: could you remove the whitespace around the equals signs?

`record_latency=False, language=None`